### PR TITLE
UI Fix: Clinical Instructions Form Styling

### PR DESCRIPTION
**Pull request by jonathanzhang53**
**GCI: Task 4 in EHR UI Grab Bag**

## Description
Link: Patient->Visits->Current->Clinical->Clinical instructions
Work: - Centralize the form. -Change tab title from “New encounter” to “Clinical instructions”

## Before
image

## After
image

### CODE REVIEW

**Comment by jonathanzhang53** [comment]
@muarachmann I don't believe that the tab title issue exists anymore.
image
image
The tab title is just the date of encounter... unless I'm in the wrong place.

**Comment by jonathanzhang53** [comment]
@muarachmann this one is ready for review.

**Comment by muarachmann** [comment]
@jonathanzhang53 i still get the title as encounter stuff. but i see the bootstrap changes

File: interface/forms/clinical_instructions/new.php [hunk 1]
```diff
@@ -51,20 +51,21 @@
...
-                <br/>
-                <input type='submit' value='<?php echo xla('Save'); ?>' class="button-css cp-submit" style="float: right">&nbsp;
-            </div>
-
-        </form>
+            <!-- Begin form -->
+            <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
+                <div class='form-group'>
+                    <textarea class="form-control" name="instruction" id ="instruction" rows=3 style='resize: vertical;'>
+                        <?php echo text($check_res['instruction']); ?>
```
Comment (muarachmann) [inline interface/forms/clinical_instructions/new.php]: Here the cursor starts at about 1/3 of the box can you please fix that (try clicking on the text area box)

Comment (jonathanzhang53) [reply interface/forms/clinical_instructions/new.php]: @muarachmann fixed. I know I'm supposed to be patient with task approval, but in light that it is the last 2 days to claim a task. Could you go to my task instance and approve it if this PR is good? https://codein.withgoogle.com/dashboard/task-instances/5694115258826752/

**Comment by jonathanzhang53** [comment]
@muarachmann got the title fixed also.

File: interface/forms/clinical_instructions/new.php [hunk 2]
```diff
@@ -51,20 +51,19 @@
...
-                    <?php echo text($check_res['instruction']); ?>
-                </textarea>
-                <br/>
-                <input type='submit' value='<?php echo xla('Save'); ?>' class="button-css cp-submit" style="float: right">&nbsp;
-            </div>
-
-        </form>
+            <!-- Begin form -->
+            <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
+                <div class='form-group'>
```
Comment (muarachmann) [inline interface/forms/clinical_instructions/new.php]: Looks awesome. Great Job
screen shot 2018-12-10 at 4 49 47 am

**Comment by Trodrige** [comment]
This looks ok with prism and tan themes but fails with light and purple themes. we have to do something about this theme issue.

**Comment by jonathanzhang53** [comment]
@Trodrige should I make changes or is this a dev issue on themes?

**Comment by Trodrige** [comment]
I am approving hoping that we find a solution to this theme issue.

**Comment by jonathanzhang53** [comment]
@Trodrige I will be available after GCI to do a few contributions. If you guys need to do some css overhaul manual labor, I can help out.

**Comment by muarachmann** [comment]
@jonathanzhang53 thanks. We need a strategy for this once done  i guess implementation wont be difficult.

**Comment by aethelwulffe** [comment]
@muarachmann PUT THIS IN YOUR THEME REVIEW ISSUE.

**Comment by aethelwulffe** [comment]
Three approvals, it is merging.  Thank you for the code and reviews.

**Comment by aethelwulffe** [comment]
Final note for history ref:  ALWAYS check demographics screen patient report tool with one of the forms in an associated encounter and look to see how it prints to PDF when changing layout on anything.

**Pull request merged**